### PR TITLE
DCCR-46: added track taxonomy terms.

### PR DESCRIPTION
**Pull request by dharizza**
Here we have the tracks taxonomy terms featurized. Please review. Thanks.

### CODE REVIEW

**Comment by kporras07** [comment]
Tracks vocabulary was already featurized in dccr_ftr_workshop.
Please fix this.

**Comment by kporras07** [comment]
@dharizza before continuing with this. Please pull master because I updated uuid_feature module.

**Comment by dharizza** [comment]
Done with your comments. Please check. Thanks.

File: sites/all/modules/features/dccr_ftr_config/dccr_ftr_config.info [hunk 1]
```diff
@@ -2,20 +2,25 @@ name = Drupal Camp Config
 core = 7.x
 package = Drupal Camp
 dependencies[] = ctools
+dependencies[] = dccr_ftr_workshop
```
Comment (kporras07) [inline sites/all/modules/features/dccr_ftr_config/dccr_ftr_config.info:5]: I really hope dccr_ftr_workshop never depends on this module ;)

**Comment by kporras07** [comment]
:shipit:

**Pull request merged**